### DISCUSSION TITLE
Sql query error after update

### DISCUSSION THREAD

**Discussion by Misiek1105** [stas-demydiuk/domoticz-zigbee2mqtt-plugin#817] [Q&A]
Hello, after upgrade zigbee2mqtt plugins I have problem with sql query error

![Screenshot_20220807-230824_Chrome](https://user-images.githubusercontent.com/95593315/183437587-827f93da-1428-48ea-8432-fbcdc7d0aefb.jpg)

Has somebody idea to resolve this problem ?

**Comment by stas-demydiuk** [comment]
Reinstalling a plugin from scratch usually solves the problem, but still not sure what's causing it. It often appears after Domoticz upgrade

**Reply by Misiek1105** [reply]
Domoticz was update 3 months ago but plugin last week.

**Reply by stas-demydiuk** [reply]
What version of the plugin were you upgrading from?

**Comment by Misiek1105** [comment]
I am not sure but 1.22 to 1.27. I used python plugins manager

**Comment by Misiek1105** [comment]
I reinstal plugin and old version domoticz and still the same. Maybe have You some idea for this? Or i must waiting to update plugin ?

**Reply by odelma** [reply]
https://github.com/stas-demydiuk/domoticz-zigbee2mqtt-plugin/issues/706#issuecomment-1368958986
